tests/rscript_http_request-vg.sh

```diff
@@ -1,3 +1,5 @@
 #!/bin/bash
 export USE_VALGRIND="YES"
+#export RS_TEST_VALGRIND_EXTRA_OPTS="--leak-resolution=high --num-callers=100 --track-origins=yes --leak-check=full"
```

Comment:
does this need to be present permanently?

Reply:
It was helpful for extending valgrind output, so it could be useful for future valgrind tests. But it can be removed.